python/cugraph/tests/utils.py

```diff
@@ -77,6 +77,12 @@
     PurePath(RAPIDS_DATASET_ROOT_DIR)/"ref/ktruss/polbooks.csv")
 ]
 
+DATASETS_TSPLIB = [
+    ["../datasets/tsplib/a280.tsp", 2579],
+    ["../datasets/tsplib/eil51.tsp", 426],
+    ["../datasets/tsplib/kroA100.tsp", 21282],
+    ["../datasets/tsplib/tsp225.tsp", 3916],
```

Comment:
Same here, we're not hardcoding relative paths anymore (see example [here](https://github.com/rapidsai/cugraph/pull/1337/files#diff-d05ca511f68d1d1443c23b813f82c300d4ede3aeb6ef5b8edc02174f149b43bfR33))

Reply:
Fixed here: https://github.com/rapidsai/cugraph/pull/1360/commits/923bd9ef94bf6fb9dee55f344379a5485c68a830